src/components/pages/public/ContactPage.vue

```diff
@@ -1,11 +1,99 @@
 <template>
-  <div>This is contact page.</div>
+  <div class="contact-page">
+    <h2 class="contact-page__heading">Contact us</h2>
+    <div class="contact-page__content-wrapper">
+      <img
+        src="@/assets/images/map.jpg"
+        class="contact-page__map"
+        alt="Cinema location on the map."
+      />
+      <div class="contact-page__contact-card">
+        <ContactCard />
+      </div>
+      <div class="contact-page__media-icons">
+        <a href="" class="contact-page__media-icon">
+          <img src=@/assets/icons/media/twitter.svg/ alt="Logo of Twitter.">
+        </a>
+        <a href="">
+          <img src=@/assets/icons/media/facebook.svg/ alt="Logo of Facebook.">
+        </a>
+        <a href="">
+          <img src=@/assets/icons/media/linkedin.svg/ alt="Logo of LinkedIn.">
+        </a>
+        <a href="">
+          <img src=@/assets/icons/media/instagram.svg/ alt="Logo of Instagram.">
+        </a>
+        <a href="">
+          <img src=@/assets/icons/media/github.svg/ alt="Logo of GitHub.">
+        </a>
+        <a href="">
+          <img src=@/assets/icons/media/behance.svg/ alt="Logo of Behance.">
+        </a>
+        <a href="">
+          <img src=@/assets/icons/media/basketball.svg alt="Logo of
+          Basketball.">
+        </a>
```

Comment:
This looks like a nice candidate to refactor considering the repetitions - maybe introduce an array of things to iterate over here with `v-for`? 💡